internal/controllers/products/get_all.go

```diff
@@ -0,0 +1,37 @@
+package products_controller
+
+import (
+	"net/http"
+
+	"github.com/maxwelbm/alkemy-g6/pkg/response"
+)
+
+func (p *ProductsDefault) GetAll() http.HandlerFunc {
```

Comment:
Vamos deixar mais simples e usar isso somente em casos necessarios
```suggestion
func (p *ProductsDefault) GetAll(w http.ResponseWriter, r *http.Request) {
```

Reply:
realmente, a gente não ta usando isso pra nada por enquanto